example_fpm.toml

```diff
@@ -0,0 +1,25 @@
+name = "package-name"
+version = "0.1.0"
+license = "BSD3"
+author = "Author name here"
+maintainer = "example@example.com"
+copyright = "2020 Author name here"
+dependencies = ["../std-lib.tar.gz"]
+compiler = "gfortran"
+devel-options = ["-g", "-Wall", "-Wextra", "-Werror", "-pedantic"]
+release-options = ["-O3"]
+
+[library]
+source-dirs = "src"
+
+[executables.executable-name]
+main = "Main.f90"
+source-dirs = "app"
+linker-options = ["-O3"]
```

Comment:
This is a compiler option, no? Probably not consequential because this is meant as example metadata. I don't know if -Ox flags can be passed to a linker or if they are just ignored. Is there a more meaningful option we can use here?

Reply:
Indeed, based on my comment above, I suggest we first tackle the common case of not specifying such options at all, but have a sane default for Debug / Release for all compilers. Exactly as Cargo works. Only later also allow to explicitly set this somehow.

Reply:
Linkers can do  some amount of optimization at link time. It's the only way to do inlining of functions from separate compilation units. This is just an example anyway, and this PR doesn't read it anyway.